app/models/restaurant.rb

```diff
@@ -23,4 +23,8 @@ class Restaurant < ApplicationRecord
 
   validates :name, presence: true
   validates :name, uniqueness: true
+
+  def active?
+    hidden == false
```

Comment:
you can write `hidden?` or it's better to rename this field to active and then write `active?`